internal/provider/resource_job_test.go

```diff
@@ -376,6 +376,44 @@ func TestAccCloudAutomatorJob_BulkDeleteImagesAction(t *testing.T) {
 	})
 }
 
+func TestAccCloudAutomatorJob_BulkDeleteEBSSnapshotsAction(t *testing.T) {
+	resourceName := "cloudautomator_job.test"
+	jobName := fmt.Sprintf("tf-testacc-job-%s", utils.RandomString(12))
+	postProcessId := acctest.TestPostProcessId()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckCloudAutomatorJobDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCloudAutomatorJobConfigBulkDeleteEBSSnapshotsAction(jobName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCloudAutomatorJobExists(testAccProviders["cloudautomator"], resourceName),
+					resource.TestCheckResourceAttr(
+						resourceName, "name", jobName),
+					resource.TestCheckResourceAttr(
+						resourceName, "action_type", "bulk_delete_ebs_snapshots"),
+					resource.TestCheckResourceAttr(
+						resourceName, "bulk_delete_images_action_value.0.exclude_by_tag_bulk_delete_ebs_snapshots", "true"),
+					resource.TestCheckResourceAttr(
+						resourceName, "bulk_delete_images_action_value.0.exclude_by_tag_key_bulk_delete_ebs_snapshots", "env"),
+					resource.TestCheckResourceAttr(
+						resourceName, "bulk_delete_images_action_value.0.exclude_by_tag_value_bulk_delete_ebs_snapshots", "production"),
+					resource.TestCheckResourceAttr(
+						resourceName, "bulk_delete_images_action_value.0.specify_base_date", "before_days"),
+					resource.TestCheckResourceAttr(
+						resourceName, "bulk_delete_images_action_value.0.before_days", "365"),
```

Comment:
[MUST]
`bulk_delete_images` になっていたので、ここだけ修正をお願いします 🙏 

```suggestion
						resourceName, "bulk_delete_ebs_snapshots_action_value.0.exclude_by_tag_bulk_delete_ebs_snapshots", "true"),
					resource.TestCheckResourceAttr(
						resourceName, "bulk_delete_ebs_snapshots_action_value.0.exclude_by_tag_key_bulk_delete_ebs_snapshots", "env"),
					resource.TestCheckResourceAttr(
						resourceName, "bulk_delete_ebs_snapshots_action_value.0.exclude_by_tag_value_bulk_delete_ebs_snapshots", "production"),
					resource.TestCheckResourceAttr(
						resourceName, "bulk_delete_ebs_snapshots_action_value.0.specify_base_date", "before_days"),
					resource.TestCheckResourceAttr(
						resourceName, "bulk_delete_ebs_snapshots_action_value.0.before_days", "365"),
```